pillarbox-player/src/main/java/ch/srgssr/pillarbox/player/analytics/metrics/MetricsCollector.kt

```diff
@@ -31,9 +30,10 @@ import java.io.IOException
  * - Dropped video frames
  * - Surface size
  */
-class MetricsCollector private constructor(
-    private val timeProvider: () -> Long,
-) {
+class MetricsCollector(
```

Comment:
Do we want to expose the constructor that takes a `timeProvider`? This should only be needed in tests.

Reply:
Yes but the constructor with custom timeProvider is never used. I can changed it to use the same Clock as the player for time management.